airgun/entities/puppet_environment.py

```diff
@@ -38,7 +38,7 @@ def delete(self, value):
         view = self.navigate_to(self, 'All')
         view.search(value)
         view.table.row(name=value)['Actions'].widget.fill('Delete')
-        self.browser.handle_alert()
```

Comment:
This really is something that should be getting handled by `browser.handle_alert` - you see it used as a kwarg to `Widget.click` within the table row too.

The PF4 dialog used for the confirmation alert isn't being recognized and handled by `widgetastic.browser.Browser.handle_alert`, selenium doesn't recognize it as an alert.

Since we have a mix of PF4 pages right now, I'm not opposed to this being a short-term solution since its being applied to the pages that are converted.

Long term we need to discuss this with the UX team (@latran, interested?) and make an update to airgun's browser class or wt.core.